Version 0.8.1

dartcafe opened the pull request:
Preparation for Version 0.8.1

splitt3r commented:
We should merge #252 first.

dartcafe commented:
Played a little bit with the line break in the description
This description 
![grafik](https://user-images.githubusercontent.com/26707476/33626965-72d4bf48-d9fc-11e7-9d54-c95f89b3b2eb.png)
gave this in the list
![grafik](https://user-images.githubusercontent.com/26707476/33626804-f4400ce6-d9fb-11e7-9ed0-e180ce33beaa.png)
![grafik](https://user-images.githubusercontent.com/26707476/33626974-77398b0e-d9fc-11e7-8154-b7a2958cfe9a.png)
and this in the polls page
![grafik](https://user-images.githubusercontent.com/26707476/33626997-7bb623fe-d9fc-11e7-961b-41c9222589fc.png)

After the commit: 
![grafik](https://user-images.githubusercontent.com/26707476/33626902-35e22652-d9fc-11e7-8b0c-06461badadf1.png)
![grafik](https://user-images.githubusercontent.com/26707476/33627006-81134430-d9fc-11e7-989e-9fb7f1a43873.png)
![grafik](https://user-images.githubusercontent.com/26707476/33627010-855e2898-d9fc-11e7-82de-14d0f3ebd370.png)
![grafik](https://user-images.githubusercontent.com/26707476/33627016-88d208b4-d9fc-11e7-8d90-7c041357edd6.png)

dartcafe commented:
CRs applied

dartcafe commented on templates/goto.tmpl.php:
```diff
@@ -81,7 +81,7 @@
 		$poll->getDescription() !== ''
 	) {
-		$description = $poll->getDescription();
+		$description = str_replace(array("\r\n", "\r", "\n") , '<br/>', htmlspecialchars($poll->getDescription()));
```
Done

splitt3r commented:
Sorry. I think i will habe some time tomorrow to do some testing.

splitt3r commented:
The avatars are only applied after refreshing the page:

![avatar](https://user-images.githubusercontent.com/1869584/33958629-3a75a7ee-e045-11e7-90d7-313a3b948aa4.png)

This looks a bit strange (deleted public poll in chrome / edge):

![grey](https://user-images.githubusercontent.com/1869584/33958742-9f7e585c-e045-11e7-98ee-4f5a7a189456.png)

Button moves to the right:

![button](https://user-images.githubusercontent.com/1869584/33958813-e3dbfed2-e045-11e7-99bb-a47dee756e11.png)

After copying the link a small notification would be nice sth. likle:

`OC.Notification.show(t('polls', 'Link copied to your Clipboard!'), {timeout: 5});`

We should fix this:

![tooltip](https://user-images.githubusercontent.com/1869584/33959002-72eb8034-e046-11e7-9cdd-49e674c04b06.png)

All in all it looks great now :) we could also fix the small things later on.

dartcafe commented:
> The avatars are only applied after refreshing the page:

Strange. Here they are applied with no problem after posting. 
![added_avatar](https://user-images.githubusercontent.com/26707476/33975954-4617f35c-e092-11e7-9a11-49de90243180.gif)

> This looks a bit strange (deleted public poll in chrome / edge):

Yup. We changed filename from polls.svg to app.svg. CSS updated

> Button moves to the right:

Create/Edit is the next project, so I will inspect later as well as #131

> After copying the link a small notification would be nice sth. like:

Changed this already in my test environment, but it was overwritten by poll. Added this again (stolen from files app 😆).

I also added a position option to the top positiond tooltips so it is diplayed under the items (I expected automatically positioning and discovered the option to define the position right now).

splitt3r commented on js/start.js:
```diff
@@ -1,6 +1,50 @@
+		if (/iPhone|iPad/i.test(navigator.userAgent)) {
+			actionMsg = t('core', 'Not supported!');
+		} else if (/Mac/i.test(navigator.userAgent)) {
+			actionMsg = t('core', 'Press ?-C to copy.');
```
This should also be ⌘-C shouldn´t it? We should only have this cliboard code once and share it between vote.js and start.js.

dartcafe replied:
I don't know, and didn't think about this too much. I just copied and pasted from files app. Actually we should think about creating more helper classes, to avoid code doubling. But I would prefer to do this in the next step.

splitt3r replied:
Ok, i´m fine with that. But the question mark should be fixed before release. In vote.js it´s ⌘-C so it should be the same here.

dartcafe replied:
Oops. Done

splitt3r commented on js/start.js:
```diff
@@ -1,6 +1,50 @@
+				$input.attr('data-original-title', t('core', 'Copy'))
+					.tooltip('fixTitle');
+			} else {
+				$input.tooltip("destroy");
```
Could you replace the double quotes here 😁

dartcafe replied:
Done.

dartcafe commented:
The time is updated live. Just wait a few seconds.

splitt3r commented:
But isn´t it possible to trigger it after posting the comment?

splitt3r commented:
Without the `width: 100%;` it looks strange in Nc 13 for me:

![width](https://user-images.githubusercontent.com/1869584/34069384-994e6000-e250-11e7-926c-2687d13bdb1d.png)

splitt3r commented:
@v1r0x should be fixed now :)

dartcafe commented:
Seems something changed between beta 1 aud beta 3. I will optimize the css after the 0.8.1 release to make this more lean.

dartcafe commented:
Any reviews ongoing?

v1r0x commented:
Yes, sorry for the long idle time. Thanks to @splitt3r  the time appears directly after sending the comment, but is in english ('now') although my language is set to german ('Gerade eben'). After a reload it is correctly translated.

Is it on purpose that the toggle icon for actions and comments is on the left side?

joergmschulz commented:
On creating a date poll, the date picker chose the current time as sample for the pickable times: 20:49 instead of 20:00
![grafik](https://user-images.githubusercontent.com/6523646/34125136-948ac05e-e435-11e7-908f-283357a6982d.png)

dartcafe commented:
@joergmschulz Yes the time picker makes nonsense. This is subject of redesign of the create page.

dartcafe commented:
@splitt3r I was wondering, that nobody mentioned the position of the details button before. 😁 It is just because the dynamycal positioning did not work as expected, when opening the sidebar. So i left it there for the moment. I will think it over later, if no one has an easy solution right now. 

The translation will work after merging to the master, I guess, when the transifex bot will run over it.

splitt3r commented:
:joy: i mentioned it somwhere but thought it was somehow intended.

Let's change now to just now. Sounds better :+1:

v1r0x commented:
> The translation will work after merging to the master, I guess, when the transifex bot will run over it.

Then I'm fine with merging. 🎉

joergmschulz commented:
With this date picker nonsense, we shouldn't merge. Which aspects should be retested manually?

dartcafe commented:
The problem with the datepicker is already present in the master branch. So the merge will have no effect regarding the picker. And this PR is big enough. We can test this with a new version of the datetimepicker after the merge, befor releasing. 

Quick view by me with version 2.5.14: this problem does not occur anymore.
xdan/datetimepicker#618

dartcafe commented:
@joergmschulz see #270
